web-access-control/web-access-control-test-manifest.ttl

```diff
@@ -26,31 +26,31 @@ manifest:protected-operation-not-read-resource-default-AWC
 manifest:protected-operation-read-resource-access-R
   a td:TestCase ;
   spec:requirementReference <https://solid.github.io/web-access-control-spec#access-modes> ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   td:preCondition "authentication", "acl" ;
   spec:testScript
     <https://github.com/solid/specification-tests/web-access-control/protected-operation/read-resource-access-R.feature> .
```

Comment:
Do we want to test for non-modification in addition to `403`? (across all tests)

Reply:
@edwardsph?

Reply:
I marked this as resolved since it is the same question as the previous comment for @kjetilk to review.

Reply:
Ah sorry, I was on a bad connection during this review; this is a duplicate.